### PR TITLE
Profiling interaction + small misc bugs

### PR DESCRIPTION
**Pull request by jaseweston**

### CODE REVIEW

File: parlai/tasks/light_dialog/worlds.py [hunk 1]
```diff
@@ -91,6 +91,7 @@ def parley(self):
             act['id'] = 'persona'
             agents[0].observe(validate(act))
         act = deepcopy(agents[0].act())
+        acts[0] = act
```
Comment (emilydinan) [inline parlai/tasks/light_dialog/worlds.py]: is this ever used anywehre?

File: parlai/scripts/profile_interactive.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        '--interactive-task',
+        type='bool',
+        default=True,
+        help='Create interactive version of task',
+    )
+    parser.set_defaults(interactive_mode=True, task='interactive')
+    return parser
+
+
+def interactive(opt, print_parser=None):
```
Comment (emilydinan) [inline parlai/scripts/profile_interactive.py]: can we rename this function to profile_interactive

File: parlai/tasks/light_dialog/worlds.py (hunk 1 above)
Comment (jaseweston) [reply parlai/tasks/light_dialog/worlds.py]: yes, it's used in the display function of the world

File: parlai/scripts/profile_interactive.py (hunk 2 above)
Comment (jaseweston) [reply parlai/scripts/profile_interactive.py]: sure

File: parlai/tasks/light_dialog/worlds.py (hunk 1 above)
Comment (emilydinan) [reply parlai/tasks/light_dialog/worlds.py]: didn't realize, thanks

File: parlai/agents/repeat_label/repeat_label.py [hunk 3]
```diff
@@ -75,5 +76,5 @@ def act(self):
                 reply['text'] = self.cantAnswerMessage
         else:
             reply['text'] = self.cantAnswerMessage
-
-        return reply
+        reply['episode_done'] = False
```
[the remark this answers is not in the document]
Comment (jaseweston) [reply parlai/agents/repeat_label/repeat_label.py]: we currently enforce models to have this or it bugs out 
we could go back and change this @stephenroller  @emilydinan  u can discuss it u guys if u like
parlai used to have all optional fields
we could out defaults in the message
or we could have defaults in the  message.get() calls
or we could leave like is

File: parlai/scripts/profile_interactive.py [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+from parlai.core.worlds import create_task
+from parlai.agents.repeat_query.repeat_query import RepeatQueryAgent
+
+import random
+import cProfile
+import io
+import pstats
+
+
+def setup_args(parser=None):
```
[the remark this answers is not in the document]
Comment (jaseweston) [reply parlai/scripts/profile_interactive.py]: i added some args actually, such as num-examples

**Pull request merged**